bofire/data_models/domain/features.py

```diff
@@ -383,6 +395,9 @@ def transform(
             elif specs[feat.key] == CategoricalEncodingEnum.DESCRIPTOR:
                 assert isinstance(feat, CategoricalDescriptorInput)
                 transformed.append(feat.to_descriptor_encoding(s))
+            elif isinstance(specs[feat.key], MolFeatures):
```

Comment:
Can you also include this in the tests?

Reply:
Testing for this can be found in `test_inputs_transform_molecular`. This only tests the transform in the forward direction. This is kept separate from `test_inputs_transform` for now because the inverse transform for molecular inputs is not implemented yet.